tests/FSharp.Data.GraphQL.IntegrationTests.Server/WebSocketMessages.fs

```diff
@@ -0,0 +1,34 @@
+namespace FSharp.Data.GraphQL.IntegrationTests.Server
```

Comment:
Same here, can we link that from StarWars?
It would be great to have integration tests for subscriptions and deferred fields. I suppose we do not have them, right?

Reply:
It looks like there's a lot we can just link over from star-wars-api, yes.  More work on this will appear in 2023/02/LHK-607-Introspection_Query .